media-gfx/{igal,pngcheck}: EAPI7, improve ebuild

mm1ke opened the pull request:
Hi,

This PR updates another few packages to EAPI7.

Please review.

mm1ke commented:
ping, just rebased this PR

Zlogene commented on media-gfx/pngcheck/pngcheck-2.3.0-r1.ebuild:
```diff
@@ -0,0 +1,29 @@
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="sys-libs/zlib"
```
not sure if it is need to stay here

mm1ke replied:
Hmm, i checked the package and it clearly rdepends on ```sys-libs/zlib```. Or, do you think it should not stay here because it's a system package? I usually only check for package which are part of followin list ```eix --system -c```
```sys-libs/zlib``` isn't included, which is why i haven't removed it.

mm1ke commented:
@Zlogene 
I've updated this PR and removed the tuxpaint-stamps change since you fixed it in the meantime with commit: 8fc754f2aefd58b69aee72f792b2d7edb2bf9777
BTW, may I notice that you could have removed the ```src_prepare``` completely and apply the patche simply via the ```PATCHES``` variable ;)

mm1ke commented:
ping, any chance we can merge this? :)
